package.json

```diff
@@ -72,6 +72,7 @@
     "dispensary": "0.10.3",
     "es6-promisify": "5.0.0",
     "eslint": "3.16.0",
+    "eslint-plugin-no-unsafe-innerhtml": "EnTeQuAk/eslint-plugin-no-unsafe-innerhtml#4b6b606d50",
```

Comment:
There is an upstream PR that needs merging first (https://github.com/mozfreddyb/eslint-plugin-no-unsafe-innerhtml/pull/16) so this will be updated once that happens.